### ISSUE TITLE
WAVFILEWARNING and USERWARNING

### ISSUE THREAD

**Issue by synkkedd** [neonbjb/tortoise-tts#452]
UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
  
  tortoise-tts\tortoise\utils\audio.py:14: WavFileWarning: Chunk (non-data) not understood, skipping it.
  sampling_rate, data = read(full_path)
  
  Anyone know a fix to these?
